google-cloud-clients/google-cloud-storage/src/test/java/com/google/cloud/storage/it/ITStorageTest.java

```diff
@@ -137,6 +165,99 @@ public static void afterClass() throws ExecutionException, InterruptedException
     }
   }
 
+  private static void prepareKmsKeys() throws IOException {
```

Comment:
Could you generally break down this method more? Maybe a function for each API call - that would make the overall flow of this method much easier to follow.